repeater/src/main/scala/com.snowplowanalytics.snowplow.storage.bigquery.repeater/Flow.scala

```diff
@@ -42,48 +41,66 @@ object Flow {
     * @param resources all application resources
     * @param events stream of events from PubSub
     */
-  def sink[F[_]: Logger: Timer: Concurrent: ContextShift](resources: Resources[F])(events: EventStream[F]): Stream[F, Unit] =
+  def sink[F[_]: Logger: Timer: Concurrent: ContextShift](
+    resources: Resources[F]
+  )(events: EventStream[F]): Stream[F, Unit] =
     events.parEvalMapUnordered(resources.concurrency) { event =>
-      val insert = checkAndInsert[F](resources.bigQuery, resources.env.config.datasetId, resources.env.config.tableId, resources.backoffTime)(event)
+      val insert = checkAndInsert[F](
+        resources.bigQuery,
+        resources.env.config.datasetId,
+        resources.env.config.tableId,
+        resources.backoffTime
+      )(event)
       resources.insertBlocker.blockOn(insert).flatMap {
         case Right(_) => resources.logInserted
+        // format: off
```

Comment:
Any reason to disable formatting here?

Reply:
Ye, it turns it into:

```
case Left(d) => resources.logAbandoned *> resources.desperates
  .enqueue1(d)
```

Reply:
That's why I prefer 140 symbols. At that point it starts to make sense to spliot expressions into separate ones:

```scala
val log = resource.logAbandoned
val enqueue = resources.desperates.enqueue1(d)
log *> enqueue
```